trk/src/Screens/ClimbCreate/index.js

```diff
@@ -62,11 +63,14 @@ const ClimbInputData = (props) => {
   };
 
   //this will later be populated by the gym documents in gym collection
-  const [gymItems, setGymItems] = useState([
-    { label: 'Palladium', value: 'Palladium' },
-    { label: 'The Cliffs LIC', value: 'The Cliffs LIC' },
-    { label: 'VITAL', value: 'VITAL' },
-    { label: 'The Gravity Vault Hoboken', value: 'The Gravity Vault Hoboken' }]);
+  const [gymItems, setGymItems] = useState([]);
+
+  useEffect(() => {
+    // Fetch gyms when component mounts
+    GymsApi().fetchGyms().then(gyms => {
+      setGymItems(gyms);
+    });
+  }, []);
```

Comment:
Using the fetchGyms function from GymsApi to access an array of all the gyms documents containing the name and id of each gym). These gyms set the gym state